lib/tasks/scheduled_ingest_bucket_deletions.rake

```diff
@@ -0,0 +1,17 @@
+namespace :scihist do
+  desc "Delete files from ingest_bucket that have been scheduled"
+  task :scheduled_ingest_bucket_delete => :environment do
+    ScheduledIngestBucketDeletion.where("delete_after < ?", Time.now).find_each do |scheduled|
```

Comment:
I feel like this should be `>`, not `<` .

Reply:
Hm.  Let me think/talk through it. 

Let's say it's Jan 1. We ingest some files, and we create ScheduledIngestBucketDeletion object for each of them with a `delete_after` of Jan 8. Meaning, it's okay to delete these after Jan 8. 

Then on Jan 9, we want to find all the `ScheduledIngestBucketDeletion` objects that have a `delete_after` *before* now, because they are safe to delete. 

So I think that is expressed by `delete_after < [now]`. 

If we were instead deleting all the objects with a `delete_after > [now]` that would mean on Jan 9 we'd be deleting an object (eg) with a delete_after of `Feb 1`, which was supposed to mean "okay delete after Feb 1". 

So I think `<` is right. What do you think?

Reply:
You are right  -- it's <. Objection withdrawn! I was really thinking in terms of a  `keep_until` date, not a `delete_after` date.